lib/MetaProcessor/MetaParser.cpp

```diff
@@ -474,6 +474,34 @@ namespace cling {
     return false;
   }
 
+  // dumps/creates a trace of the requested representation.
+  bool MetaParser::istraceCommand() {
+    if (getCurTok().is(tok::ident) &&
+        getCurTok().getIdent().equals("trace")) {
+      consumeToken();
+      skipWhitespace();
+      if (!getCurTok().is(tok::ident))
+          return false;
+      std::string ident = getCurTok().getIdent();
+      consumeToken();
+      skipWhitespace();
+      if (getCurTok().is(tok::eof)) {
+        m_Actions->actOnstatsCommand(ident=="ast" 
+            ? llvm::StringRef("asttree") : llvm::StringRef(ident), llvm::StringRef());
+        consumeToken();
+      }
+      else if (getCurTok().is(tok::ident)) {
+        m_Actions->actOnstatsCommand(ident=="ast"
+            ? llvm::StringRef("asttree") : llvm::StringRef(ident), 
```

Comment:
Ditto. And this seems to be a replication, can you combine these two if branches with an `||`?